puppet/zulip/files/supervisor/conf.d/thumbor.conf

```diff
@@ -0,0 +1,12 @@
+[program:zulip-thumbor]
+command=/home/zulip/deployments/current/zulip-thumbor-venv/bin/thumbor --port=9995 --conf=/home/zulip/deployments/current/zthumbor/thumbor.conf
```

Comment:
I think this is going to be problematic since I have given commands using hard wired path to current directory which might not always be the directory from which app is deployed.

Reply:
oops, I think this is just fine. We do symlink to current at deployment.